docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php

```diff
@@ -54,6 +54,34 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
         'cta_button' => $this->t('CTA'),
       ],
     ];
+    $element['background'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Background Color'),
+      '#description' => $this->t('Choose a background for the component.'),
+      '#empty_option' => $this->t('- Change the background -'),
+      '#default_value' => $paragraph->getBehaviorSetting('sul_teaser_styles', 'background'),
+      '#options' => [
+        'black' => $this->t('Black'),
+      ],
+    ];
+    $element['background_sprinkles'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Background Sprinkles'),
+      '#description' => $this->t('Choose the position of the "sprinkles".'),
+      '#empty_option' => $this->t('- Change the position -'),
+      '#default_value' => $paragraph->getBehaviorSetting('sul_teaser_styles', 'background_sprinkles'),
+      '#options' => [
+        'top_left' => $this->t('Top Left'),
+        'top_right' => $this->t('Top Right'),
+        'bottom_left' => $this->t('Bottom Left'),
+        'bottom_right' => $this->t('Bottom Right'),
+      ],
+      '#states' => [
+        'invisible' => [
+          'select[name="behavior_plugins[sul_teaser_styles][background]"]' => ['value' => ''],
```

Comment:
I'm not sure if this is the reason why the state doesn't work. You'll need to review the functionality.
```suggestion
          'select[name="behavior_plugins[sul_card_styles][background]"]' => ['value' => ''],
```